controller/node_controller.go

```diff
@@ -571,6 +571,26 @@ func (nc *NodeController) getDiskInfoMap(node *longhorn.Node) map[string]*diskIn
 	return result
 }
 
+// Check multiple disks belongs to the same filesystem ID
+// Skip the ones are not in ready status
+func (nc *NodeController) checkSameFsidInDisks(id string, disks []string, diskStatusMap map[string]*types.DiskStatus) bool {
```

Comment:
The function name is confusing. `check` doesn't say if we expect true or false if there is same FSID among the disks.

Use `isFSIDDuplicatedWithExistingReadyDisk(name string, ...)` instead. FSID is an abbreviation so it needs to be capitalized. Also, use `name` instead of `id`, since it can be mistaken for the `fsid` in this case. Same with the variable used in the loop.

Also, update the comment to indicate that all the disks passed in here have the same FSID.